scripts/mkcloud

```diff
@@ -774,9 +776,57 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
```

Comment:
Just tested this and this line failed from missing .artifacts dir. Jenkins might create one, but for manual tests it is not there.